examples/gym_ex/proxy/env.py

```diff
@@ -115,7 +119,10 @@ def reset(self) -> None:
         """
         if not self._agent.multiplexer.is_connected:
             self._connect()
-        gym_msg = GymMessage(performative=GymMessage.Performative.RESET)
+        gym_msg = GymMessage(
+            dialogue_reference=self._dialogues.new_self_initiated_dialogue_reference(),
+            performative=GymMessage.Performative.RESET,
+        )
```

Comment:
Gym dialogue structure is as follows:

```
reply:
  reset: [status]
  status: [act, close, reset]
  act: [percept]
  percept: [act, close, reset]
  close: []
```
you need to ensure this is followed here.

Reply:
So after sending `reset` we expect the next message to be a `status`; etc.

Reply:
need to discuss